tools/validate_paths.go

```diff
@@ -189,12 +191,28 @@ func checkFiles(knownOC map[string]pathType, files []string) ([]file, error) {
 		}
 
 		var errs []string
+		var dependencies []string
 
 		// Unmarshal will report syntax errors (although generally without line numbers).
 		if err := prototext.Unmarshal(bs, &tmp); err != nil {
 			errs = append(errs, err.Error())
 		}
 
+		// Validate feature profile ID name by checking path 
+		targetFeatureProfileName := getFeatureProfileNameFromPath(f, tmp)
+		featureProfileIDName := tmp.GetId().GetName()
+		validProfile[featureProfileIDName] = true
+		if targetFeatureProfileName != featureProfileIDName {
+			errs = append(errs, featureProfileIDName+" is inconsistent with path")
```

Comment:
should include what we expect the target to be, like `featureProfileIDName+" is inconsistent with path, want "+targetFeatureProfileName`